shapely/_geometry.py

```diff
@@ -677,11 +709,14 @@ def get_precision(geometry, **kwargs):
 
     Examples
     --------
-    >>> get_precision(Geometry("POINT (1 1)"))
+    >>> from shapely import Point
+    >>> point = Point(1, 1)
+    >>> get_precision(point)
     0.0
-    >>> geometry = set_precision(Geometry("POINT (1 1)"), 1.0)
+    >>> geometry = set_precision(point, 1.0)
     >>> get_precision(geometry)
     1.0
+    >>> import numpy as np
     >>> np.isnan(get_precision(None))
```

Comment:
In this case we could maybe actually also do something more simple like:

```
>>> get_precision(None)
nan
```

(since np.nan actually prints a value in contrast to `None`, we don't necessarily have to do the workaround)